ci/deploy-console-plugins.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+script_dir=$(dirname "$0")
+
+echo ""
+echo "deploy console plugins (forklift, minivirt)"
+kubectl apply -f ${script_dir}/yaml/minivirt-plugin.yaml
+cat ${script_dir}/yaml/forklift-plugin.yaml.template | sed "s/{{FORKLIFT_IMAGE_REPOSITORY}}/quay.io\/kubev2v/g" | kubectl apply -f -
```

Comment:
Lots of options here.  

Could do an envvar definition with the "use what is already defined or "quay.io/kubev2v" if it is not already defined style:
```
FORKLIFT_IMAGE_REPOSITORY=${FORKLIFT_IMAGE_REPOSITORY:=quay.io/kubev2v}
```


Then, I kinda like the idea of `envsubst` to push env vars into a file:
```suggestion
envsubst < ${script_dir}/yaml/forklift-plugin.yaml.template | kubectl apply -f -
```


Or maybe a different way with sed:
```suggestion
sed "s/{{FORKLIFT_IMAGE_REPOSITORY}}/$FORKLIFT_IMAGE_REPOSITORY/g" \
  ${script_dir}/yaml/forklift-plugin.yaml.template | kubectl apply -f -
```


Maybe helm?  @yaacov would probably know better about that option

Reply:
I like the `sed` approach, but I'm are missing 2 layers of abstraction:
a. that task should be runnable from a ci script ( `bash /ci/run-e2e-tests.sh` )
b. the task should be runnable from `package.jjson` npm script ( `npm run e2e-tests` )
c. the github action run the `npm` script

Reply:
1. The placeholders syntax (`{{ FOO }}`) seems quite popular - see Helm, Go tempating, jinja2. The `sed` is used here as simple replacement for a full templating engine (at this point we don't need an extra power )
2. the alternative would be using Kustomize